apps/cowswap-frontend/src/legacy/state/orders/orderPrice.test.ts

```
@@ -23,14 +30,29 @@ describe('getEstimatedExecutionPrice()', () => {
+    const orderWithoutFee = getLimitOrderWithFee(sellAmountBeforeFee, buyAmount, zeroFee)
     const executionPriceWithoutFee = getEstimatedExecutionPrice(orderWithoutFee, fillPrice, zeroFee.toString())
 
     expect(executionPriceWithFee?.toFixed(12)).toEqual('1820.007835117276')
     expect(executionPriceWithoutFee?.toFixed(12)).toEqual('1820.000000000000')
   })
 })
+
+describe('getRemainderAmountsWithoutSurplus()', () => {
+  it('Should return the remainder amounts without the surplus', () => {
+    const order = getLimitOrderWithFee(
+      6.42619072256475129773952e23,
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 642619072256475100000000

(lint/correctness/noPrecisionLoss)